cookbooks/cdo-nginx/test/cookbooks/nginx_test/templates/default/init.d.erb

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+set -e
+# Example init script, this can be used with nginx, too,
+# since nginx and unicorn accept the same signals
+
+# Feel free to change any of the following variables for your app:
+TIMEOUT=${TIMEOUT-60}
+APP_ROOT=<%= @app_root %>
+PID=<%= @pid_file %>
+CMD="cd $APP_ROOT; LANG=en_US.UTF-8 bundle exec unicorn -D -c <%= @src_file %> -E <%= @env %>"
```

Comment:
yeah, this was a copy/paste of the [init.d.erb](https://github.com/code-dot-org/code-dot-org/blob/cdo-nginx/cookbooks/cdo-apps/templates/default/init.d.erb) template sysvinit script in `cdo-apps`, which itself was originally derived from [`examples/init.sh`](http://bogomips.org/unicorn.git/tree/examples/init.sh) upstream. I'll rename these templates and add comments to clarify